hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/upgrade/UpgradeTestIT.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2012 - 2020 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.splicemachine.derby.impl.sql.execute.upgrade;
+
+import com.splicemachine.access.hbase.HBasePartitionAdmin;
+import com.splicemachine.db.impl.sql.catalog.SYSNATURALNUMBERSRowFactory;
+import com.splicemachine.derby.test.framework.SpliceSchemaWatcher;
+import com.splicemachine.derby.test.framework.SpliceUnitTest;
+import com.splicemachine.derby.test.framework.SpliceWatcher;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.client.Admin;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.ConnectionFactory;
+import org.junit.*;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+
+/**
+ * Note: These tests need to be excluded in platform_it/pom.xml,
+ * as they are only mean to be run with the check_upgrade.sh script.
+ *
+ * e.g.
+ * bash pipelines/upgrade-testing/check_upgrade.sh 3.1.0.2000 -T 4
```

Comment:
I ran this command, but -T option is not accepted by ./start-splice-cluster

./start-splice-cluster: illegal option -- T

Reply:
yes, -T was only introduced in 3.x.0.2001, so this doesn't work.